src/pudl/transform/eia176.py

```diff
@@ -359,3 +349,134 @@ def core_eia176__yearly_gas_disposition_by_consumer(
     df = df.dropna(subset=["consumers", "revenue", "volume_mcf"], how="all")
 
     return df
+
+
+@asset(io_manager_key="pudl_io_manager")
+def core_eia176__yearly_gas_disposition(
+    _core_eia176__yearly_company_data: pd.DataFrame,
+    core_pudl__codes_subdivisions: pd.DataFrame,
+    raw_eia176__continuation_text_lines: pd.DataFrame,
+) -> pd.DataFrame:
+    """Produce company-level gas disposition (EIA176, Part B)"""
+    extras = ["operating_state"]
+
+    keep = [
```

Comment:
When available, it would be helpful to include the appropriate line number in the field description, so that a user can conclusively trace the reported value back to where it would have appeared in a filing.